mne/io/brainvision/brainvision.py

```diff
@@ -513,9 +535,40 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                 if lp_s:
                     info['lowpass'] = 1. / info['lowpass']
         else:
-            info['lowpass'] = np.min(np.array(lowpass, dtype=np.float))
-            warn('Channels contain different lowpass filters. Lowest filter '
-                 'setting will be stored.')
+            heterogeneous_lp_filter = True
+            if lp_s:
+                # We convert channels with disabled filters to having
+                # infinitely relaxed / no filters
+                lowpass = [float(_) if _ not in ('NaN', 'Off') else 0.0
+                           for _ in lowpass]
+                info['lowpass'] = np.max(np.array(lowpass, dtype=np.float))
+                try:
+                    info['lowpass'] = 1. / info['lowpass']
+                except ZeroDivisionError:
```

Comment:
In particular, I went with the try/except because it should be a rather unusual condition that  heterogeneously set lowpass filters have their strongest filter be "no filter" -- that would mean that some channels had 'Off' and some had 'NaN' and none had anything besides those two. I can't imagine BV produced BV files actually containing that, but it's an anomalous situation that we can recover from, hence the try/except bit.